Library/Tracking/KSRAnalytics.swift

```diff
@@ -1810,9 +1810,13 @@ private func userProperties(for user: User?, config _: Config?, _ prefix: String
   var props: [String: Any] = [:]
 
   props["backed_projects_count"] = user?.stats.backedProjectsCount
-  props["created_projects_count"] = user?.stats.createdProjectsCount
+  props["created_projects_count"] = (user?.stats.createdProjectsCount ?? 0) +
+    (user?.stats
+      .draftProjectsCount ??
+      0) // Stats.createdProjectsCount is the count of projects user has lauched only, while event property `created_projects_count` includes Stats.createdProject + Stats.draftProjectsCount
   props["is_admin"] = user?.isAdmin
-  props["launched_projects_count"] = user?.stats.memberProjectsCount
+  props["launched_projects_count"] = user?.stats
+    .createdProjectsCount // event property`launched_projects_count` = Stats.createdProjectsCount
```

Comment:
change to:

product and insights defines `launched_projects_count` as only the `createdProjectsCount`